stl/inc/xfilesystem_abi.h

```diff
@@ -309,6 +310,12 @@ _NODISCARD __std_win_error __stdcall __std_fs_create_symbolic_link(
 _NODISCARD __std_win_error __stdcall __std_fs_read_reparse_data_buffer(_In_ __std_fs_file_handle _Handle,
     _Out_writes_bytes_(_Buffer_size) void* _Buffer, _In_ unsigned long _Buffer_size) noexcept;
 
+_NODISCARD __std_win_error __stdcall __std_fs_write_reparse_data_buffer(
+    _In_ __std_fs_file_handle _Handle, _In_ const __std_fs_reparse_data_buffer* const _Buffer) noexcept;
+
+[[nodiscard]] bool __stdcall __std_fs_is_junction_from_reparse_data_buffer(
+    _In_ const __std_fs_reparse_data_buffer* const _Buffer) noexcept;

```

Comment:
In function declarations that aren't definitions, we conventionally omit top-level `const` on parameters (because it doesn't matter - the compiler immediately ignores it - and it is not relevant to the caller). This is one of the few ways that declarations regularly vary from definitions. Occurs for both of these `_Buffer` parameters.